stdlib/json.gr

```diff
@@ -0,0 +1,2155 @@
+      }
+    }
+  } else {
+    // And this is just the combination of both flags, which means
+    // doing almost the same as for the case above for
+    // escapeNonASCII=true, but also escape the ASCII control codepoint
+    // 127 (Delete).
+    (codePoint: Number) => {
+      if (
+        codePoint > 31 &&
+        codePoint != 0x0022 &&
+        codePoint != 0x005C &&
+        codePoint < 127
+      ) {
+        // fast path for chars that never need any escaping
+        addCharFromCodePoint(codePoint, buffer)
+      } else {
+        emitEscapedCodePoint(codePoint, buffer)
+      }
+    }
+  }
+
+  let emitEscapedQuotedString = if (!format.escapeHTMLUnsafeSequences) {
+    (s: String) => {
+      Buffer.addChar('"', buffer)
+
+      // Note that it's important for performance that the closure passed to forEachCodePoint
+      // is not allocated inline here, but just once when creating the writer.
+
+      String.forEachCodePoint(emitCodePoint, s)
+
+      Buffer.addChar('"', buffer)
+    }
+  } else {
+    // Special handling for the escapeHTMLUnsafeSequences flag.
+    // Escaping a sequence requires keeping track of previous characters,
+    // which is difficult and suboptimal when using a function to iterate
+    // the input string. So we don't want to pay the price in other cases.
+    // This cannot be done just in the emitCodePoint function.
+    // It could be possible to implement more optimally, but would
+    // complicate things even more than this.
+    (s: String) => {
+      Buffer.addChar('"', buffer)
+
+      let mut prevCodePoint = 0
+
+      String.forEachCodePoint(codePoint => {
+        if (codePoint == 47) {
+          if (format.escapeHTMLUnsafeSequences && prevCodePoint == 60) {
+            Buffer.addChar('\\', buffer)
+            Buffer.addChar('/', buffer)
+          } else {
+            // otherwise just emit the slash as-is
+            addCharFromCodePoint(codePoint, buffer)
+          }
+        } else {
+          emitCodePoint(codePoint)
+        }
+
+        prevCodePoint = codePoint
+      }, s)
+
+      Buffer.addChar('"', buffer)
+    }
+  }
+
+  let implHelper = {
+    format,
+    bufferPretty: buffer,
+    emitEscapedQuotedStringPretty: s => {
+      emitEscapedQuotedString(s)
+    },
+    printNewLine,
+    printIndentation,
+  }: JsonWriterPrettyImplHelper
+
+  {
+    emit: json => {
+      let error = printElementPretty(json, implHelper, 0)
+      if (Option.isSome(error)) return error
+      if (format.finishWithNewLine) printNewLine()
+      return None
+    },
+  }: JsonWriter
+}
+
+/**
+ * Prints the JSON object into a string with specific formatting settings.
+ *
+ * @param json: The JSON object to print
+ * @param format: formatting option
+ * @returns A `Result` object with either the string containing the printed JSON or an error if the input object cannot be represented in the JSON format.
+ *
+ * @example
+ * print(
+ *   Result.unwrap(
```

Comment:
Since we are using `print` for the examples, why do we use`unwrap`? Can't be just show `Ok({"currency":"\u20ac","price":99.9}")` is printed?